### DISCUSSION TITLE
MyTest Discussion

### DISCUSSION THREAD

**Discussion by OsagieOkunbor** [Cyfrin/foundry-full-course-cu#1575] [General]
I have just started my Solidity programming journey. Wish me luck

**Comment by EngrPips** [comment]
Hey @OsagieOkunbor , i wish you best of luck !

**Reply by OsagieOkunbor** [reply]
Thank you so much

**Reply by EngrPips** [reply]
Anytime friend

**Comment by Harshvardhantomar** [comment]
Best of luck @OsagieOkunbor.
